src/js/player.js

```diff
@@ -664,11 +687,14 @@ class Player extends Component {
    * @param {number} [value]
    *        The value to set the `Player`'s width to.
    *
+   * @param {boolean} [skipListeners]
+   *        Skip the playerresize event trigger
+   *
    * @return {number}
    *         The current width of the `Player` when getting.
    */
-  width(value) {
-    return this.dimension('width', value);
+  width(value, skipListeners) {
```

Comment:
hmmm why is this different then master?

Reply:
huh, weird. Maybe my local master checkout had an old version or something when I merged. I'll take a look.

Reply:
Nope, I think the merge is failing and thinking that we want to keep this change when we reverted it in master. Lame. I don't really want to run a rebase. I'll probably cherry-pick the revert or something.